code/modules/admin/verbs/request_internet_sound.dm

```diff
@@ -2,6 +2,10 @@
 	set category = "OOC"
 	set name = "Request Internet Sound"
 
+	if(usr == null)
+		to_chat(src, span_boldwarning("An Error Occured, this might be because round just started. If so try again."), confidential = TRUE)
```

Comment:
```suggestion
		to_chat(src, span_boldwarning("An error occurred, this might be because the round just started. If so, try again."), confidential = TRUE)
```